docs/operators.md

```diff
@@ -54,7 +49,11 @@ package main
 import (
 	"context"
 	"fmt"
-	
+	"os"
```

Comment:
The code updates here are a drive-by as the example still used `simple.NewOperator`. This whole doc may also need some more updates--while it is factually correct, it may not be a good first-brush resource for the operator pattern, as it talks about the `operator` package construction of an operator, which we now hide behind `simple.App`. I added a quick into paragraph to add that context because I'm not sure if/when/how much we want to update this doc.